src/Components/PaperEditor/TranscriptsContainer/TranscriptMenu/index.js

```diff
@@ -8,70 +8,72 @@ import ButtonGroup from 'react-bootstrap/ButtonGroup';
 import DropdownButton from 'react-bootstrap/DropdownButton';
 import LabelsList from '../LabelsList/index.js';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faHighlighter, faCog } from '@fortawesome/free-solid-svg-icons';
+import { faCog, faTag } from '@fortawesome/free-solid-svg-icons';
+import './index.scss';
 
 const TranscriptMenu = (props) => {
   const labels = props.labels;
 
-  const HighlightButton = (
+  const getSelectedLabelColour = (transcriptLabels) => {
```

Comment:
I'm not sure if it's this function, but initially when I load my workspace, the colour of the active label flickers from orange to yellow, or sometimes to a different colour. It might be that this should be some sort of useEffect hook, so the colour is a state variable and it only gets set once transcriptLabels load? I'm sure there are other ways to address it too

Reply:
Okay so I've done a real makeover on my previous iteration and tried to use state and useEffect this time – let me know if you think it's working any better – I think it is 🤞